templates.dist/estate/map/map-google.php

```diff
@@ -28,22 +28,20 @@
 use onOffice\WPlugin\EstateList;
 use onOffice\WPlugin\ViewFieldModifier\EstateViewFieldModifierTypes;
 
-$pEstatesClone = clone $pEstates;
-
-/* @var $pEstatesClone EstateList */
-$pEstatesClone->resetEstateIterator();
+/* @var $pEstates EstateList */
```

Comment:
I guess this change leads back on the fact that I started developing these default views a couple of months ago. From my point of view it can be reverted.

Reply:
still open? can you first clone the object and then use resetEstateIterator() on it, please?

Reply:
Commited!